config/patches/openssl/openssl-1.0.2zi-freebsd-nocryptodev.patch

```diff
@@ -0,0 +1,11 @@
+--- crypto/engine/eng_cryptodev.c.orig	2017-07-06 01:00:00 UTC
++++ crypto/engine/eng_cryptodev.c
+@@ -35,7 +35,7 @@
+ #if (defined(__unix__) || defined(unix)) && !defined(USG) && \
+         (defined(OpenBSD) || defined(__FreeBSD__))
+ # include <sys/param.h>
+-# if (OpenBSD >= 200112) || ((__FreeBSD_version >= 470101 && __FreeBSD_version < 500000) || __FreeBSD_version >= 500041)
++# if (OpenBSD >= 200112) || ((__FreeBSD_version >= 470101 && __FreeBSD_version < 500000) || (__FreeBSD_version >= 500041 && __FreeBSD_version < 1300000))
```

Comment:
Can we put justification for this < 13.0.0 bounding in the PR and then probably also on a second or later line for the ultimate commit message?

Reply:
This https://forums.freebsd.org/threads/upgraded-freebsd-12-3-to-13-1-has-not-cryptodev-module-in-boot-loader-conf-file.86770/ seems to imply people can just load the module for /dev/crypto.

I'm worried about dropping things that might end up like the Debian openssh patch that inadvertently hurt security for years.

Reply:
Pil and I talked today. I see what needs to happen here. I'll update the ticket to reflect this: